src/CMakeLists.txt

```diff
@@ -280,7 +280,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 elseif(${CMAKE_SYSTEM_NAME} MATCHES "Emscripten")
   set(LEANC_STATIC_LINKER_FLAGS "-lleancpp -lInit -lStd -lLean -lnodefs.js -lleanrt")
 else()
-  set(LEANC_STATIC_LINKER_FLAGS "-Wl,--start-group -lleancpp -lInit -lStd -lLean -Wl,--end-group")
+  set(LEANC_STATIC_LINKER_FLAGS "-Wl,--start-group -lleancpp -lInit -lStd -lLean -lleanrt -Wl,--end-group")
```

Comment:
@leodemoura By the way, I took this change as an opportunity to look at the dependencies between our packages/static libraries again: there are *almost* two separate SCCs Init/Std/leanrt (runtime) and Lean/leancpp, except for the following edges from the former to the latter:
* Init depends on leancpp because of `lean_name_eq`
* leanrt depends on leancpp because of the `util/{buffer.h,*_ref.h}` headers. Not sure if we should just move them into the runtime. Note that they are header-only except for the single method `object_ref::mk_cnstr`.

Reply:
I will try to break these dependencies today.

Reply:
BTW, as far as I can tell the only runtime file that uses  `*_ref.h` files is `process.cpp`. We should be able to avoid them there.

Reply:
I considered that too, but I *really* put them to use there. It's possible, but might end up white ugly.